exchanges/exchange.go

```diff
@@ -788,27 +793,22 @@ func (e *Base) SetAPIURL() error {
 	var err error
 	e.Config.API.OldEndPoints = nil
 	if e.Config.API.Endpoints != nil {
-		for a, meow := range e.Config.API.Endpoints {
-			if meow == "" || meow == "NON_DEFAULT_HTTP_LINK_TO_EXCHANGE_API" || meow == "NON_DEFAULT_HTTP_LINK_TO_WEBSOCKET_EXCHANGE_API" {
+		for key, val := range e.Config.API.Endpoints {
+			if val == "" ||
+				val == "NON_DEFAULT_HTTP_LINK_TO_EXCHANGE_API" ||
+				val == "NON_DEFAULT_HTTP_LINK_TO_WEBSOCKET_EXCHANGE_API" {
```

Comment:
Can export a config const so when we deprecate this it's more obvious to remove here as well.